bpf/bpf_lxc.c

```diff
@@ -525,7 +525,7 @@ static __always_inline int handle_ipv4_from_lxc(struct __ctx_buff *ctx,
 	__u32 __maybe_unused tunnel_endpoint = 0;
 	__u8 __maybe_unused encrypt_key = 0;
 	__u32 monitor = 0;
-	__u8 reason;
+	__u8 ct_ret;
```

Comment:
Should we also rename in other functions (IPv6, bpf_host, etc.) for consistency?

I'm also wondering if this change should be in its own PR to *not* backport it as it may cause a lot of conflicts.

Reply:
> Should we also rename in other functions (IPv6, bpf_host, etc.) for consistency?

Yes, I was thinking about doing it in a separate PR as it's a bit out of scope with the changes proposed here

> I'm also wondering if this change should be in its own PR to not backport it as it may cause a lot of conflicts.

I'll probably take care of backporting this PR anyway since there are conflicts also in the first commit, so I think it's fine